packages/harmony/src/theme.css

```
@@ -68,47 +68,74 @@
+    --chart-4: 43 80% 48%;
+    --chart-5: 271 50% 55%;
+    --sidebar: 150 12% 94.5%;
+    --sidebar-foreground: 168 12% 16%;
+    --sidebar-primary: 142.13 76.22% 36.27%;
+    --sidebar-primary-foreground: 0 0% 100%;
+    --sidebar-accent: 150 10% 92%;
+    --sidebar-accent-foreground: 168 12% 16%;
+    --sidebar-border: 150 9% 88%;
+    --sidebar-ring: 142.13 76.22% 36.27%;
     --font-sans:
       'Resist Sans Text', ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto,
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)